Resources/Prototypes/Entities/Clothing/OuterClothing/wintercoats.yml

```diff
@@ -0,0 +1,413 @@
+  components:
+  - type: Sprite
+    sprite: Clothing/OuterClothing/WinterCoats/coatatmos.rsi
+  - type: Clothing
+    sprite: Clothing/OuterClothing/WinterCoats/coatatmos.rsi
+  - type: TemperatureProtection
+    coefficient: 0.1
+  - type: Armor
+    modifiers:
+      coefficients:
+        Slash: 0.95
+        Heat: 0.75
+
+- type: entity
+  parent: ClothingOuterBase
+  id: ClothingOuterWinterCap
+  name: captain's winter coat
+  description: A heavy jacket made from 'synthetic' animal furs. 
+  components:
+  - type: Sprite
+    sprite: Clothing/OuterClothing/WinterCoats/coatcap.rsi
+  - type: Clothing
+    sprite: Clothing/OuterClothing/WinterCoats/coatcap.rsi
+  - type: TemperatureProtection
+    coefficient: 0.1
+  - type: Armor
+    modifiers:
+      coefficients:
+        Slash: 0.95
+        Heat: 0.75
+
+- type: entity
+  parent: ClothingOuterBase
+  id: ClothingOuterWinterCargo
+  name: cargo winter coat
+  description: A heavy jacket made from 'synthetic' animal furs. 
+  components:
+  - type: Sprite
+    sprite: Clothing/OuterClothing/WinterCoats/coatcargo.rsi
+  - type: Clothing
+    sprite: Clothing/OuterClothing/WinterCoats/coatcargo.rsi
+  - type: TemperatureProtection
+    coefficient: 0.1
+  - type: Armor
+    modifiers:
+      coefficients:
+        Slash: 0.95
+        Heat: 0.75
+
+- type: entity
+  parent: ClothingOuterBase
+  id: ClothingOuterWinterCE
+  name: ce's winter coat
+  description: A heavy jacket made from 'synthetic' animal furs. 
+  components:
+  - type: Sprite
+    sprite: Clothing/OuterClothing/WinterCoats/coatce.rsi
+  - type: Clothing
+    sprite: Clothing/OuterClothing/WinterCoats/coatce.rsi
+  - type: TemperatureProtection
+    coefficient: 0.1
+  - type: Armor
+    modifiers:
+      coefficients:
+        Slash: 0.95
+        Heat: 0.75
+
+- type: entity
+  parent: ClothingOuterBase
+  id: ClothingOuterWinterCentcom
+  name: centcom winter coat
+  description: A heavy jacket made from 'synthetic' animal furs. 
+  components:
+  - type: Sprite
+    sprite: Clothing/OuterClothing/WinterCoats/coatcentcom.rsi
+  - type: Clothing
+    sprite: Clothing/OuterClothing/WinterCoats/coatcentcom.rsi
+  - type: TemperatureProtection
+    coefficient: 0.1
+  - type: Armor
+    modifiers:
+      coefficients:
+        Slash: 0.95
+        Heat: 0.75
+
+- type: entity
+  parent: ClothingOuterBase
+  id: ClothingOuterWinterChem
+  name: chemistry winter coat 
+  description: A heavy jacket made from 'synthetic' animal furs. 
+  components:
+  - type: Sprite
+    sprite: Clothing/OuterClothing/WinterCoats/coatchem.rsi
+  - type: Clothing
+    sprite: Clothing/OuterClothing/WinterCoats/coatchem.rsi
+  - type: TemperatureProtection
+    coefficient: 0.1
+  - type: Armor
+    modifiers:
+      coefficients:
+        Slash: 0.95
+        Heat: 0.75
+
+- type: entity
+  parent: ClothingOuterBase
+  id: ClothingOuterWinterCMO
+  name: cmo's winter coat
+  description: A heavy jacket made from 'synthetic' animal furs. 

```

Comment:
If you optionally want to add these somewhere, they could go into the head of staff lockers, I'm sure they'll be around for a long while. Otherwise still fine.

Reply:
I put those in the HoP uniform printer to give it some more purpose since it's not used much atm.